### ISSUE TITLE
Using the buffer not working

### ISSUE THREAD

**Issue by corte** [frinyvonnick/node-html-to-image#13]
In the readme says that we can get the buffer using:
```javascript
const image = await nodeHtmlToImage({
    html: "<html><body>HELLO</body></html>"
});
```

But that throws exception immediately:
```
(node:22148) UnhandledPromiseRejectionWarning: Error: You must provide an output property.
    at module.exports (C:\Workspace\Personal\html-calendar\node_modules\node-html-to-image\src\index.js:17:11)
    at getImageBuffer (C:\Workspace\Personal\html-calendar\html-image.js:17:18)
    at Object.<anonymous> (C:\Workspace\Personal\html-calendar\html-image.js:8:5)
    at Module._compile (internal/modules/cjs/loader.js:1151:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1171:10)
    at Module.load (internal/modules/cjs/loader.js:1000:32)
    at Function.Module._load (internal/modules/cjs/loader.js:899:14)
    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:71:12)
    at internal/main/run_main_module.js:17:47
(node:22148) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 2)
(node:22148) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

**Comment by frinyvonnick** [comment]
Hi @corte,

Thank you for taking time to post this issue 🙏. It is duplicated, I close it in favor of #12, the author of this issue will open a pull request.

**Comment by stefanocudini** [comment]
I have same problem! how can I solve it?

**Comment by frinyvonnick** [comment]
@stefanocudini which version of node-html-to-image do you use? This issue has been fixed in [2.0.0](https://github.com/frinyvonnick/node-html-to-image/releases/tag/v2.0.0). Can you share your code so I could reproduce it?
